gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HookHandler.java

```diff
@@ -1238,6 +1239,7 @@ private void handleListenerRegistration(final HttpServerRequest request) {
                         RequestLogger.logRequest(vertx.eventBus(), request, status, buffer);
                     }
                     vertx.eventBus().publish(SAVE_LISTENER_ADDRESS, listenerStorageUri);
+                    trackableEventPublish.publish(SAVE_LISTENER_ADDRESS, listenerStorageUri);
```

Comment:
I assume it is by intent to DUPLICATE that publication here for whatever reason? All other places do a REPLACE of the call. Except this one, which now does TWO calls.
```patch
     vertx.eventBus().publish(SAVE_LISTENER_ADDRESS, listenerStorageUri);
+    trackableEventPublish.publish(SAVE_LISTENER_ADDRESS, listenerStorageUri);
```

Reply:
Thanks, I removed it already, but don't know it doesn't marked it as outdated